.github/workflows/release-create.yml

```diff
@@ -0,0 +1,167 @@
+
+# Creates a release tag for the current in-tree version from the main or another branch.
+
+# This workflow creates the git commits + git tag for a Iceberg Catalog Migrator release.
+
+name: Create Release
+
+on:
+  # Manually triggered
+  workflow_dispatch:
+    inputs:
+      releaseFromBranch:
+        description: 'The branch name the release from, leave empty to release from latest commit on main.'
+        required: false
+      bumpType:
+        description: 'Optional: bump patch, minor or major version (`patch`, `minor`, `major`). Default is `minor`.'
+        required: true
+        type: string
+        default: "minor"
+
+jobs:
+  create-release:
+    name: Create release
+    runs-on: ubuntu-22.04
+    if: github.repository_owner == 'projectnessie'
+    env:
+      RELEASE_FROM: ${{ github.event.inputs.releaseFromBranch }}
+      BUMP_TYPE: ${{ github.event.inputs.bumpType }}
+
+    steps:
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: 11
+
+      - name: Checkout
+        uses: actions/checkout@v3.5.2
+        with:
+          ref: ${{ env.RELEASE_FROM }}
+          fetch-depth: '0'
+
+      - name: Bump to release version
+        uses: gradle/gradle-build-action@v2
+        with:
+          cache-disabled: true
+          arguments: :bumpVersion --bumpType ${{ env.BUMP_TYPE }} --bumpToRelease
+
+      - name: Get release version
+        run: |
+          RELEASE_VERSION=$(cat version.txt)
+          LAST_TAG=$(git describe --abbrev=0 --tags --match=catalog-migrator-* || echo "catalog-migrator-0.1.0")
+
+          echo "LAST_TAG=${LAST_TAG}" >> ${GITHUB_ENV}
+          echo "RELEASE_VERSION=${RELEASE_VERSION}" >> ${GITHUB_ENV}
+          echo "GIT_TAG=catalog-migrator-${RELEASE_VERSION}" >> ${GITHUB_ENV}
+
+          cat <<! >> $GITHUB_STEP_SUMMARY
+          ## About to release
+
+          Version information right before the Git release tag and commit:
+
+          | Name | Value |
+          | --- | --- |
+          | Iceberg Catalog Migrator release version  | ${RELEASE_VERSION}            | 
+          | Git tag name            | \`catalog-migrator-${RELEASE_VERSION}\` | 
+          | Previous Git tag        | \`${LAST_TAG}\`               | 
+          | Release from branch     | ${RELEASE_FROM}               | 
+          | Bump type               | ${BUMP_TYPE}                  |
+          | Before release Git HEAD | \`$(git rev-parse HEAD)\`     |
+          !
+
+      - name: Update .md files referencing latest version
+        run: |
+          LAST_VERSION="${LAST_TAG#catalog-migrator-}"
+          LAST_VERSION_PATTERN="$(echo ${LAST_VERSION} | sed 's/\([^.]*\)[.]\([^.]*\)[.].*/\1\\.\2/')\\."
+
+          for fn in SECURITY.md README.md ; do
+            sed -i "s/${LAST_VERSION_PATTERN}[^ ]*/${RELEASE_VERSION}/" ${fn}
+          done
+
+      - name: Configure release-bot-user in git config
```

Comment:
I expected a new account here as CEL java has its own release-bot. 

https://github.com/projectnessie/cel-java/blob/13bf72f6adc0cee7db5125a43e459a77472b35ec/.github/workflows/release.yml#L72-L75

It is just me figuring out whether each project has to have its own release or can it be reused.  Based on this I will assume, it can be reused. Thanks.